composer.json

```diff
@@ -38,5 +38,17 @@
         "branch-alias": {
             "dev-master": "7.0.x-dev"
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^8.1",
```

Comment:
As we minimum required PHP 7.1, we can't use PHPUnit 8. Can you downgrade to PHPUnit 7, please?

Reply:
Sure :man_facepalming: my bad

Reply:
did that